lambda/slack.js

```diff
@@ -6,13 +6,7 @@ const token = process.env.SLACK_API_TOKEN
 const web = new WebClient(token)
 
 function send(channel, message) {
-  web.chat.postMessage(channel, message, function(err, res) {
-    if (err) {
-      console.log('Error:', err)
-    } else {
-      console.log('Message sent: ', res)
-    }
-  })
+  return web.chat.postMessage(channel, message)
```

Comment:
does that work? :)

Reply:
Well, yes and no. The Slack SDK indeed supports promises as well (found the info hidden in [a ticket](https://github.com/slackapi/node-slack-sdk/issues/139#issuecomment-192748795)).

But when I double checked I realised I screwed up the function definition on line 14 of `index.js`. I'll fix that.